support-frontend/assets/helpers/abTests/__tests__/abtestTest.ts

```diff
@@ -337,69 +337,193 @@ describe('init', () => {
-					excludeCountriesSubjectToContributionsOnlyAmounts: true,
+					excludeContributionsOnlyCountries: false,
 				}),
 			};
 
 			const participations: Participations = abInit({
 				...abtestInitalizerData,
 				abTests,
-				selectedAmountsVariant,
+				selectedAmountsVariant: buildSelectedAmountsVariant(
+					contributionsOnlyAmountsTestName,
+				),
 			});
 
-			expect(participations).toEqual({});
+			expect(participations.t1).toBeDefined();
+		});
+
+		it(`does assign a user to the test if excludeContributionsOnlyCountries is true BUT selectedAmountsVariant test name is NOT ${contributionsOnlyAmountsTestName}`, () => {
+			const abTests = {
+				t1: buildTest({
+					variants: [
+						buildVariant({ id: 'control' }),
+						buildVariant({ id: 'variant' }),
+					],
+					excludeContributionsOnlyCountries: true,
+				}),
+			};
+
+			const participations: Participations = abInit({
+				...abtestInitalizerData,
+				abTests,
+				selectedAmountsVariant: buildSelectedAmountsVariant('foo'),
+			});
+
+			expect(participations.t1).toBeDefined();
+		});
+	});
+
+	describe('includeOnlyCountriesSubjectToContributionsOnlyAmounts', () => {
+		it(`does assign a user to a test if audience is CONTRIBUTIONS_ONLY and selectedAmountsVariant test name is ${contributionsOnlyAmountsTestName}`, () => {
+			const abTests = {
+				t1: buildTest({
+					variants: [
+						buildVariant({ id: 'control' }),
+						buildVariant({ id: 'variant' }),
+					],
+					audiences: {
+						CONTRIBUTIONS_ONLY: {
+							offset: 0,
+							size: 1,
+						},
+					},
+					excludeContributionsOnlyCountries: false,
+				}),
+			};
+
+			const participations: Participations = abInit({
+				...abtestInitalizerData,
+				abTests,
+				selectedAmountsVariant: buildSelectedAmountsVariant(
+					contributionsOnlyAmountsTestName,
+				),
+			});
+			expect(participations.t1).toBeDefined();
+		});
+
+		it(`does assign a user to a test if excludeContributionsOnlyCountries is false, includeOnlyContributionsOnlyCountries is false and selectedAmountsVariant test name is ${contributionsOnlyAmountsTestName}`, () => {
+			const abTests = {
+				t1: buildTest({
+					variants: [
+						buildVariant({ id: 'control' }),
+						buildVariant({ id: 'variant' }),
+					],
+					excludeContributionsOnlyCountries: false,
+				}),
+			};
+
+			const participations: Participations = abInit({
+				...abtestInitalizerData,
+				abTests,
+				selectedAmountsVariant: buildSelectedAmountsVariant(
+					contributionsOnlyAmountsTestName,
+				),
+			});
+			expect(participations.t1).toBeDefined();
 		});
 
-		it(`does assign a user to a test if excludeCountriesSubjectToContributionsOnlyAmounts is false and selectedAmountsVariant test name is ${contributionsOnlyAmountsTestName}`, () => {
+		it(`does not assign a user to a test if excludeContributionsOnlyCountries is true, includeOnlyContributionsOnlyCountries is false and selectedAmountsVariant test name is ${contributionsOnlyAmountsTestName}`, () => {
```

Comment:
This test also appears to be the same as https://github.com/guardian/support-frontend/pull/6695/files#diff-ea6fffa4fcd929f02fe9c5d08ac836f2e162cd156cf20495c3656938686dc841R341 in the `excludeContributionsOnlyCountries` describe block.